ci-operator/jobs/openshift/cincinnati-graph-data/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- crawford
+- lucab
+- steveeJ
+- wking
+- vrutkovs
```

Comment:
same here.

Reply:
> same here

https://github.com/openshift/release/pull/4958#discussion_r366380380 and openshift/cincinnati-graph-data#19 will get this addressed via the automatic `OWNERS` bumps after this PR lands.